scripts/install.sh

```diff
@@ -17,16 +17,25 @@ if test "$INSTALL_PACKAGES" = "true"; then
     set -u
 fi
 
-# ensure groups
-for group in developers researchers reviewers; do
-    if ! getent group $group > /dev/null; then
-        echo "Adding group $group"
-        groupadd $group
+ensure_group() {
+    local name=$1
+    shift;
+    if ! getent group "$name" > /dev/null; then
```

Comment:
one of the most incredibly unnecessary habits I picked up from the ruby style guide  https://github.com/rubocop/rubocop was a completely irrational aversion to `if not` where there is an `else` branch... I can't pass by this block without suggesting.... wdyt?

```bash
if getent group "$name" > /dev/null; then
        groupmod "$@" "$name"
    else
        groupadd "$@" "$name"
    fi
```